sql/log.cc

```diff
@@ -10345,6 +10345,7 @@ binlog_background_thread(void *arg __attribute__((unused)))
   thd->store_globals();
   thd->security_ctx->skip_grants();
   thd->set_command(COM_DAEMON);
+  THD_count::count--;
```

Comment:
It seems a fair amount of the `binlog_background_thread` initialization is also done in `sql_class.cc::create_background_thd()` . Would it be more consistent to use that as an initialization for the `BBT`?

Reply:
[This is just a test comment, please ignore]